Modules/Core/Common/include/itkGaussianOperator.hxx

```diff
@@ -59,19 +53,21 @@ GaussianOperator<TPixel, VDimension, TAllocator>::GenerateCoefficients() -> Coef
     }
   }
   // Normalize the coefficients so that their sum is one.
-  for (it = coeff.begin(); it < coeff.end(); ++it)
+  for (typename CoefficientVector::iterator it = coeff.begin(); it < coeff.end(); ++it)
   {
     *it /= sum;
   }
```

Comment:
Maybe a range-based for loop:

    for (auto & coefficient: coeff)
    {
      coefficient /= sum;
    }

Reply:
After this set of refactorings, there is a cling-tidy filter that will make this type of change automatically across the toolkit.

It is one of the primary motivations for making these intermediary refactorings.